anonymize/anonymize_test.go

```diff
@@ -0,0 +1,45 @@
+package anonymize
+
+import (
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type AnonymizeSuite struct {
+	suite.Suite
+}
+
+func TestAnonymizeSuite(t *testing.T) {
+	suite.Run(t, new(AnonymizeSuite))
+}
+
+func (s *AnonymizeSuite) TestAnonymizeWithNoSecretsReturnsUnchangedString()  {
+	anonimized, err := Anonymize("Hello Dear User", "")
+	s.Nil(err)
```

Comment:
Small suggestion: `s.Require().Nil(...)` so if it's suddenly broken, the tests fail early instead of panicking